src/audio/drc/testbench_drc_sin/typdef.h

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+#pragma once
+#include <stdint.h>
+typedef signed char int8_t;
+typedef unsigned char uint8_t;
+typedef short int16_t;
+typedef unsigned short uint16_t;
+typedef int int32_t;
+typedef unsigned int uint32_t;
+
+#define SINE_C_Q20 341782638 /* 2*SINE_NQUART/pi in Q12.20 */
+#define SINE_NQUART 512 /* Must be 2^N */
+#define SINE_TABLE_SIZE (SINE_NQUART + 1)
+
+/*define macros*/
+#define Q_CONVERT_FLOAT(f, qy) \
```

Comment:
this is a duplicate file that you already included once. Please dedup your code amongst your commit and against the project itself.